datadog_cluster_agent/datadog_checks/datadog_cluster_agent/check.py

```diff
@@ -0,0 +1,46 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import OpenMetricsBaseCheck
+
+
+class DatadogClusterAgentCheck(OpenMetricsBaseCheck):
+    """
+    Collect Cluster Agent metrics from its Prometheus endpoint
+    """
+
+    def __init__(self, name, init_config, instances):
+        metrics = {
+            'admission_webhooks_certificate_expiry': 'admission_webhooks.certificate_expiry',
+            'admission_webhooks_mutation_attempts': 'admission_webhooks.mutation_attempts',
+            'admission_webhooks_reconcile_success': 'admission_webhooks.reconcile_success',
+            'admission_webhooks_webhooks_received': 'admission_webhooks.webhooks_received',
+            'cluster_checks_busyness': 'cluster_checks_busyness',
+            'cluster_checks_configs_dangling': 'cluster_checks_configs_dangling',
+            'cluster_checks_configs_dispatched': 'cluster_checks_configs_dispatched',
+            'cluster_checks_nodes_reporting': 'cluster_checks_nodes_reporting',
+            'cluster_checks_rebalancing_decisions': 'cluster_checks_rebalancing_decisions',
+            'cluster_checks_rebalancing_duration_seconds': 'cluster_checks_rebalancing_duration_seconds',
+            'cluster_checks_successful_rebalancing_moves': 'cluster_checks_successful_rebalancing_moves',
+            'cluster_checks_updating_stats_duration_seconds': 'cluster_checks_updating_stats_duration_seconds',
+            'external_metrics_delay_seconds': 'external_metrics_delay_seconds',
+            'external_metrics_processed_value': 'external_metrics_processed_value',
+            'rate_limit_queries_limit': 'rate_limit_queries_limit',
+            'rate_limit_queries_period': 'rate_limit_queries_period',
+            'rate_limit_queries_remaining': 'rate_limit_queries_remaining',
+            'rate_limit_queries_reset': 'rate_limit_queries_reset',
```

Comment:
why do we have a `admission_webhooks.*` sub-namespace but not `cluster_checks.*` `external_metric.*` `rate_limit_queries.*` ?